### PR TITLE
Updated freedesktop.org

### PR DESCRIPTION
**Pull request by fbernhart**
freedesktop-dot-org

**Issue:** #3169 
**Alexa rank:** -

### Checklist
  - [ ] I updated the JSON data in `_data/simple-icons.json` --> (wasn't necessary)
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
I took the already used SVG.
I used Inkscape to fix size and center
I used https://jakearchibald.github.io/svgomg/ to optimize the resulting SVG

### CODE REVIEW

File: icons/freedesktop-dot-org.svg [hunk 1]
```diff
@@ -1 +1 @@
...
...-.7756.044-1.1972.1386L4.4308 6.8164c-2.4282.7063-3.1071 2.0162-2.6242 4.2652l1.7495 6.7045c.4829 2.1136 2.2831 3.1013 4.373 2.4397l11.6622-3.0427c2.1228-.5042 3.1071-2.0966 2.6242-4.2652L20.466 6.2134c-.3924-1.6763-1.5495-2.5718-3.1759-2.5783zm1.2512 3.5528c.2161-.0126.413.1263.4703.3457l.6228 2.388v.0005l.0108.0421c.0654.2502-.0796.4981-.33.5636l-3.4046.8882c-.2508.0653-.4986-.0799-.564-.3306l-.6337-2.43c-.0655-.2508.0795-.4986.3306-.5641l3.4045-.8882a.4794.4794 0 01.0933-.0151zM8.325 7.847c.3002-.017.575.1763.6542.48l.777 2.978c.0906.3467-.113.6935-.46.7841l-4.1724 1.0888c-.3473.0906-.6938-.1126-.7842-.4594l-.777-2.9785a.6377.6377 0 01.4594-.7842l4.173-1.0883a.6654.6654 0 01.13-.0204zm5.8578 1.4664l.173.6633-4.1206.4751-.1732-.6633zm.8995 2.5406c.184.147.4105.234.6498.248L14.158 14.471c-.1354-.1347-.3108-.226-.501-.2545a.8718.8718 0 00-.1402-.007zm-5.627.8952l2.0315 1.9608-.5193.1354h-.0005c-.0796.0208-.1516.055-.2195.095l-2.0623-1.99zm4.1062 2.1377c.0816.0109.15.0688.1726.1559v.0005l.4244 1.6265.0108.0421c.0304.1165-.034.2274-.1505.2578l-2.3378.6094c-.1163.0303-.2268-.0342-.2572-.151l-.4358-1.6686c-.0303-.1163.0343-.2267.151-.2572l2.3378-.61a.2189.2189 0 01.0847-.0054z"/></svg>
```
Comment (adamrusted) [inline icons/freedesktop-dot-org.svg:1]: There are some ***very*** slight differences between the PR path (red) and source at 7 decimals (black).
image

Could you try and rebuild to 3 decimals again, see if it changes anything?
If not, @simple-icons/maintainers are we okay with these *slight* differences?

**Review by adamrusted — CHANGES_REQUESTED** [review]

File: icons/freedesktop-dot-org.svg (hunk 1 above)
Comment (ericcornelissen) [reply icons/freedesktop-dot-org.svg:1]: I think as long as they can be avoided at a [precision of 5 or less](https://github.com/simple-icons/simple-icons/blob/5e4b4d66672208daceb77bf077fdfa03afa58e4b/.svglintrc.js#L14) any differences with an official vector version should be avoided.

Comment (fbernhart) [reply icons/freedesktop-dot-org.svg]: I can see the same slight diffferences. After some messing around with the icon a bit, it seems like it's Inkscape's fault, converting some strokes to paths that is causing this.

Unless someone else wants to give it a try with Illustrator etc, we probably have to stick with it.

Comment (adamrusted) [reply icons/freedesktop-dot-org.svg]: > Unless someone else wants to give it a try with Illustrator etc, we probably have to stick with it.

I can take a quick look in Illustrator once I've ran through the backlog of updated icons 😅

Comment (adamrusted) [reply icons/freedesktop-dot-org.svg]: Can another @simple-icons/maintainers please review this with the latest push?

Comment (fbernhart) [reply icons/freedesktop-dot-org.svg]: @adamrusted Looks good to me! Better than my version. 💯

**Pull request merged**